pytorch_lightning/strategies/deepspeed.py

```diff
@@ -741,7 +741,7 @@ def save_checkpoint(self, checkpoint: Dict, filepath: _PATH) -> None:
             )
         # Use deepspeed's internal checkpointing function to handle partitioned weights across processes
         # dump states as a checkpoint dictionary object
-        _exclude_keys = ["state_dict", "optimizer_states", "lr_schedulers"]
+        _exclude_keys = ["state_dict", "optimizer_states"]
```

Comment:
If the `lr_schedulers` weren't getting saved, why didn't this test fail? It checks that the `lr_schedulers` key exists in the saved checkpoint

https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/models/test_hooks.py#L496

Reply:
Ah, I didn't know this hook test uses deepspeed. Thanks! 

It's because the entry is a list and it is empty if no scheduler is defined/saved. And so this matches ANY.

Reply:
and `on_save_checkpoint` where the test asserts the contents gets called before the `save_checkpoint` call (where deepspeed dropped the key)